engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/bulk/content-hosts-bulk-module-streams-modal.controller.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * @ngdoc object
+ * @name  Bastion.content-hosts.controller:ContentHostsBulkModuleStreamsModalController
+ *
+ * @requires $scope
+ * @requires $http
+ * @requires $location
+ * @requires $window
+ * @requires $timeout
+ * @requires $uibModalInstance
+ * @requires HostBulkAction
+ * @requires HostCollection
+ * @requires ModuleStream
+ * @requires Nutupane
+ * @requires CurrentOrganization
+ * @requires Notification
+ * @requires BastionConfig
+ * @requires hostIds
+ * @requires translate
+ *
+ * @description
+ *   Provides the functionality for the content host module streams list and actions.
+ */
+
+angular.module('Bastion.content-hosts').controller('ContentHostsBulkModuleStreamsModalController',
+    ['$scope', '$http', '$location', '$window', '$timeout', '$uibModalInstance', 'HostBulkAction',
+     'HostCollection', 'ModuleStream', 'Nutupane', 'CurrentOrganization', 'Notification', 'BastionConfig',
+     'hostIds', 'translate',
+    function ($scope, $http, $location, $window, $timeout, $uibModalInstance, HostBulkAction, HostCollection,
+              ModuleStream, Nutupane, CurrentOrganization, Notification, BastionConfig, hostIds, translate) {
+        var nutupaneParams;
+
+        console.log(hostIds);
+
+        $scope.ok = function () {
+            $uibModalInstance.close();
+        };
+
+        $scope.cancel = function () {
+            $uibModalInstance.dismiss('cancel');
+        };
+
+        $scope.moduleStreamActions = [
+          { action: 'enable', description: translate("Enable")},
+          { action: 'disable', description: translate("Disable")},
+          { action: 'install', description: translate("Install")},
+          { action: 'update', description: translate("Update")},
+          { action: 'remove', description: translate("Remove")}
+        ];
```

Comment:
should we have an angular service for DRYing this up?

Reply:
yes, was planning on that